tests/README.md

```diff
@@ -0,0 +1,427 @@
+# Automated regression tests for librdkafka
+
+
+## Supported test environments
+
+While the standard test suite works well on OSX and Windows,
+the full test suite (which must be run for PRs and releases) will
+only run on recent Linux distros due to its use of ASAN, Kerberos, etc.
+
+
+## Automated broker cluster setup using trivup
+
+A local broker cluster can be set up using
+[trivup](https://github.com/edenhill/trivup), which is a Python package
+available on PyPi.
+These self-contained clusters are used to run the librdkafka test suite
+on a number of different broker versions or with specific broker configs.
+
+trivup will download the specified Kafka version into its root directory,
```

Comment:
side note: this is broken on my machine for older versions not kept on mirrors. At a glance, the code in trivup looks like it was written to try a mirror first then the main archive, but it's not working - it's giving a not a tgz archive, so it's probably getting back a 404 page or something.

Reply:
That should be fixed on the master version of trivup.